test/components/isEvolved.ts

```diff
@@ -0,0 +1,54 @@
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+it("Properly evolves a Pokemon at exactly its level requirement", (): void => {
+    // Arrange
+    const fsp: FullScreenPokemon = stubBlankGame();
+    const pokemonTitle: string[] = ["C", "H", "A", "R", "M", "A", "N", "D", "E", "R"];
+
+    // Act
+    fsp.itemsHolder.setItem("PokemonInParty", [
+        fsp.equations.newPokemon(pokemonTitle, 15)
```

Comment:
15 is a bit of an arbitrary number. Can you compute these using storage, so that it doesn't matter which Pokemon title is being passed?